src/errors/RequestResponseError.ts

```diff
@@ -7,12 +7,13 @@ import { ErrorResponse } from "../api/RestAPI";
 type ErrorRequest = Omit<ErrorResponse, "status">;
 
 const serializeErrorResponse = ({ code, httpCode, errors }: ErrorRequest): string => {
-    const formattedErrors = (errors || []).filter(Boolean).map((error) => {
+    // (pH) FIXME: `errors sometimes is not an array. Fix type and maybe the cause of it
+    const formattedErrors = (errors || [])?.filter(Boolean).map((error) => {
```

Comment:
Somehow the `errors` array sometimes is not an array... I will investigate the types of the errors at one point. I think it became more and more messy and the type was not updated accordingly.